docs/public-networks/reference/cli/options.md

```diff
@@ -2856,9 +2856,9 @@ Using pruning with [private transactions](../../../private-networks/concepts/pri
 
 :::
 
-:::note
+:::caution
 
-Pruning is being deprecated for [Bonsai Tries](../../concepts/data-storage-formats.md#bonsai-tries) and is currently not being updated.
+Forest pruning (using the `--pruning-enabled` option) is deprecated and will be removed in a future release. We recommend using [Bonsai Tries](../../concepts/data-storage-formats.md#bonsai-tries) as an alternative for saving disk space.
```

Comment:
```suggestion
Forest pruning is deprecated and will be removed in a future release. We recommend using [Bonsai Tries](../../concepts/data-storage-formats.md#bonsai-tries) as an alternative for saving disk space.
```

Don't need the "(using the `--pruning-enabled` option)" parenthetical here since note is directly in the `--pruning-enabled` option description.

Reply:
That's fine!

Reply:
updated. Thank you :)